scripts/extensions/sams/src/components/assets/assetEditor.tsx

```diff
@@ -141,6 +169,42 @@ class AssetEditorComponent extends React.PureComponent<IProps> {
                         </FormRow>
                     </FormGroup>
                 )}
+                {this.fieldEnabled('tags') && (
+                    <FormGroup>
+                        <FormRow>
+                            <Autocomplete
+                                key={nextId.name}
+                                label={gettext('Tags')}
+                                value={''}
+                                keyValue="name"
+                                items={[]}
+                                search={(searchString, callback) => {
+                                    let cancelled = false;
+
+                                    httpRequestJsonLocal<IAutoTaggingSearchResult>({
```

Comment:
This might be better off in `src/api/assets.ts`, and accessed in UI components through the api

`samsApi.assets.searchTags(searchString)`